roles/install_operator_prereqs/tasks/main.yml

```diff
@@ -69,9 +88,12 @@
     when:
       - offline_cataloger_help_result.rc is defined
       - offline_cataloger_help_result.rc != 0
-  when:
```

Comment:
@mvalahtv, please check if you didn`t remove some condition which is needed for openshift

Reply:
In our case it will just install offline catalgue to image. I think it can be there. Later even openshift can use our image